assets/src/blocks/CarouselHeader/ArrowsAndIndicators.js

```diff
@@ -0,0 +1,24 @@
+import { Arrows } from './Arrows';
+import { Indicators } from './Indicators';
+
+export function ArrowsAndIndicators({
```

Comment:
Seems more straightforward to use `Arrows` and `Indicators` directly? That's already a bit less props to pass on.

Reply:
Indeed. Sometimes you don't know where to stop, my biggest fear in JSX is creating a `<Div>` component. 😛